pam/integration-tests/cmd/exec-client/client.go

```diff
@@ -0,0 +1,287 @@
+//go:build pam_tests
```

Comment:
This file is locked under a build tag, so it should be transparent on the filename IMHO.

Reply:
You mean, should I call it differently?

Reply:
The file, yes.

Reply:
Mh, so maybe changing the tag too and keeping the `exec` thingy?

I was using a more generic build tag put probably we can be more speciric.

Reply:
I'm fine with both (either renaming the tag or the file). But it's good to indicate that this file won't be compiled by default.